docker/Dockerfile

```diff
@@ -29,12 +29,10 @@ RUN apk add --no-cache --virtual .build_deps \
  && mkdir -p /synapse/cache \
  && pip install -f /synapse/cache --upgrade --process-dependency-links . \
  && mv /synapse/docker/start.py /synapse/docker/conf / \
- && rm -rf \
-        setup.cfg \
-        setup.py \
-        synapse \
+ && rm -rf /synapse \
```

Comment:
If we remove this line then this will reduce the size a little without doing deletes that aren't worthwhile.